lib/node_modules/@stdlib/math/base/special/maxabs/lib/main.js

```diff
@@ -22,28 +22,22 @@
 
 var abs = require( '@stdlib/math/base/special/abs' );
 var max = require( '@stdlib/math/base/special/maxn' );
```

Comment:
```suggestion
var max = require( '@stdlib/math/base/special/max' );
```

We can use the binary interface for this implementation.